File: Desktop/TecnPoint/TecnPoint.Modelos/DTO/AtualizaChamadoDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TecnPoint.Modelos.Enum;

namespace TecnPoint.Modelos.DTO
{
    public class AtualizaChamadoDTO
    {
        public long idChamado { get; set; }
        public PrioridadeChamado? prioridade { get; set; }
        public StatusChamado? status { get; set; }
        public long? idUsuario { get; set; }
        public long? idModulo { get; set; }
        public long? idJornada { get; set; }

        public AtualizaChamadoDTO() { }

        public AtualizaChamadoDTO(long idChamado, PrioridadeChamado prioridade, StatusChamado status, long idUsuario, long idModulo, long idJornada)
        {
            this.idChamado = idChamado;
            this.prioridade = prioridade;
            this.status = status;
            this.idUsuario = idUsuario;
            this.idModulo = idModulo;
            this.idJornada = idJornada;
        }
    }
}

File: Backend/tecnpoint/src/main/java/com/techsolutions/tecnpoint/entities/DTO/AtualizaChamadoDTO.java
package com.techsolutions.tecnpoint.entities.DTO;

import com.techsolutions.tecnpoint.entities.enums.PrioridadeChamado;
import com.techsolutions.tecnpoint.entities.enums.StatusChamado;
import lombok.Data;

@Data
public class AtualizaChamadoDTO {
    PrioridadeChamado prioridade;
    StatusChamado status;
    Long idUsuario;
    Long idModulo;
    Long idJornada;
}
